tools/cloud-build/project-cleanup-slurm.yaml

```diff
@@ -35,8 +35,8 @@ steps:
             sleep $wait
         fi
 
-        # look only for tests that either use Slurm5, Slurm6 or Filestore
-        builds_filter="tags=m.schedmd-slurm-gcp-v6-controller OR tags=m.schedmd-slurm-gcp-v5-controller OR tags=m.filestore"
+        # look only for tests that either use Slurm5, or Slurm6
```

Comment:
I think we may want to eventually tag based on compact placement vs "Slurm". In the meantime, I think you should add a comment explaining why we are filtering here on slurm here:

`v5: clean project metadata, v5+v6: clean resource policies"

(above nit, but choose your own wording and merge)